addons/website/static/src/components/configurator/configurator.xml

```diff
@@ -110,13 +110,15 @@
                             </div>
                         </div>
                         <div t-if="state.recommendedPalette" class="w-75 mx-auto px-2 pt-3" style="max-width: 184px;">
-                            <h6 class="text-center text-success d-block badge mb-0 mt-n2">Detected Colors</h6>
                             <div t-attf-class="palette_card rounded-pill overflow-hidden d-flex {{getters.getSelectedPaletteName() == 'recommendedPalette' ? 'selected' : ''}}"
                                  t-on-click="selectPalette('recommendedPalette')" t-attf-style="background-color: {{state.recommendedPalette.color3}}">
                                 <div class="color_sample w-100" t-attf-style="background-color: {{state.recommendedPalette.color1}}"/>
                                 <div class="color_sample w-100" t-attf-style="background-color: {{state.recommendedPalette.color3}}"/>
                                 <div class="color_sample w-100" t-attf-style="background-color: {{state.recommendedPalette.color2}}"/>
                             </div>
+                            <button class="btn btn-primary text-nowrap mt-3 d-flex mx-auto" t-on-click="selectPalette('recommendedPalette')">
+                                Let's go! <i class="fa fa-angle-right text-white-50 my-auto pl-2" title="dropdown_angle_right" role="img"/>
+                            </button>
```

Comment:
- d-block instead of d-flex (or maybe I don't see why you would want to use d-flex?)
- my-auto useless (was only useful to counter the use of d-flex)
- Maybe want to use pl-1 instead of pl-2 or remove the space after "Let's go!" (which now appears since not using d-flex)
- Why would you want to show "dropdown_angle_right" as text to the user? (I am really curious how you went and typed that here :thinking:)

Reply:
> my-auto useless (was only useful to counter the use of d-flex)

Indeed needed when using d-flex but since d-flex doesn't need to be used...

> Why would you want to show "dropdown_angle_right" as text to the user? (I am really curious how you went and typed that here thinking)

Just a stupid copy/paste of what is done on the other chevron in the description screen. I'll remove this attribute here and on the other chevron.